src/components/media/controls/commonControls.tsx

```diff
@@ -170,60 +143,75 @@ export function VolumeSettings(props: MediaControls) {
         aria-valuetext={`Volume: ${volumeFraction * percentMultiple}%`}
         value={volumeFraction}
         onChange={handleVolumeChange}
+        sx={{ color: props.color }}
       />
     </Box>
   )
 }
 
-export function TranscriptToggle(props: TranscriptToggleProps) {
-  const Icon = props.isTranscriptShown
-    ? KeyboardArrowUpIcon
-    : KeyboardArrowDownIcon
+export function CaptionsToggle(props: Toggle) {
```

Comment:
I figured since we're using it in both the `Sound` and `Video` components for now until the `Video` settings menu design flow was finalized, since it seems like playback rate, captions, etc will later be handled there.